tests/framework/CodeInterfaceTests/AcceleratedCFD/test_perturb_acdf.xml

```diff
@@ -0,0 +1,145 @@
+<Simulation verbosity='debug'>
+  <!-- test info block -->
+
+  <TestInfo>
+      <name>framework/CodeInterfaceTests/AcceleratedCFD.perturbACFD1</name>
+      <author>alfoa</author>
+      <created>2020-11-03</created>
+      <classesTested>Models.Code.AcceleratedCFD</classesTested>
+      <description>
+          An example of using the AcceleratedCFD code interface.
+          This test is aimed to test the mechanics of the interface (no executable).
+      </description>
+  </TestInfo>
+
+  <!-- test info block end -->
+
+  <RunInfo>
+    <WorkingDir>AcceleratedCFDTest1</WorkingDir>
+    <Sequence>testAcceleratedCFD,dumpResults</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <Files>
+    <Input name="AcceleratedCFDinput" type="input">podInputs.xml</Input>
+    <Input name="Cx" type="mesh-x">Cx</Input>
+    <Input name="Cy" type="mesh-y">Cy</Input>
+    <Input name="Cz" type="mesh-z">Cz</Input>
+  </Files>
+
+  <Models>
+	<Code name="AcceleratedCFDCode" subType="AcceleratedCFD">
+        <executable>./romRun</executable>
+        <outputLocations>
+            <coordinates>
+             (0.0,0.0,0.0) (1.0,1.0,1.0) (min,min,min) (max,max,max) (middle,middle,middle)
+            </coordinates>
+        </outputLocations>
+        
+    </Code>
+  </Models>
+
+  <Distributions>
+    <UniformDiscrete name='numModes'>
+      <lowerBound>10</lowerBound>
+      <upperBound>30</upperBound>
+      <strategy>withoutReplacement</strategy>
+    </UniformDiscrete>
+  </Distributions>
+
+  <Samplers>
+    <MonteCarlo  name="MC_samp">
+      <samplerInit>
+        <limit>2</limit>
+      </samplerInit>
+      <variable name='velMods'>
+        <distribution>numModes</distribution>
+      </variable>
+      <variable name='scalMods'>
+        <distribution>numModes</distribution>
+      </variable>
+    </MonteCarlo>
+  </Samplers>
+
+  
+  
+  <DataObjects>
+    <PointSet name="AcceleratedCFDPointSet">
+      <Input>velMods,scalMods</Input>
+      <Output>time,volVectorField-0_0_0_0_0_0-x,
+      volVectorField-0_0_0_0_0_0-y,
+      volVectorField-0_0_0_0_0_0-z,
+      volVectorField-1_0_1_0_1_0-x,
+      volVectorField-1_0_1_0_1_0-y,
+      volVectorField-1_0_1_0_1_0-z,
+      volVectorField-min_min_min-x,
+      volVectorField-min_min_min-y,
+      volVectorField-min_min_min-z,
+      volVectorField-max_max_max-x,
+      volVectorField-max_max_max-y,
+      volVectorField-max_max_max-z,
+      volVectorField-middle_middle_middle-x,
+      volVectorField-middle_middle_middle-y,
+      volVectorField-middle_middle_middle-z
+      </Output>
+    </PointSet>
+    <HistorySet name="AcceleratedCFDHistorySet">
+      <Input>velMods,scalMods</Input>
+      <Output>time,volVectorField-0_0_0_0_0_0-x,
```

Comment:
should time be the pivot param?

Reply:
yes, our default pivot param is 'time', and our code will try to remove index from the output.